Interfaces/API/Dirac.py

```diff
@@ -75,6 +81,39 @@ def __init__( self, withRepo = False, repoLocation = '', useCertificates = False
 
     self.__clients = None
 
+  def __checkFileArgument( self, fnList, prefix = None, single = False ):
```

Comment:
It is not the same, here it just checks the arguments and refuses in some cases to accept a list because the method cannot accept a list...